packages/hash/frontend/src/blocks/userBlocks.tsx

```diff
@@ -68,7 +75,7 @@ const mergeBlocksData = (
   });
 };
 
-export const UserBlocksProvider: React.FC<{ value: UserBlocks }> = ({
+export const UserBlocksProvider: FC<{ value: UserBlocks }> = ({
```

Comment:
I don’t think we’ll be able to use `FC` in Node 18 typings – they remove implicit children there. Something to keep in mind, no need to change anything in this PR.

Reply:
Good, that's a sensible change.